src/Microsoft.AspNet.Identity/SignInManager.cs

```diff
@@ -197,34 +198,32 @@ public virtual void SignOut()
                 throw new ArgumentNullException(nameof(user));
             }
 
-            using (await BeginLoggingScopeAsync(user))
+            var error = await PreSignInCheck(user);
+            if (error != null)
             {
-                var error = await PreSignInCheck(user);
-                if (error != null)
-                {
-                    return Logger.Log(error);
-                }
-                if (await IsLockedOut(user))
-                {
-                    return Logger.Log(SignInResult.LockedOut);
-                }
-                if (await UserManager.CheckPasswordAsync(user, password))
-                {
-                    await ResetLockout(user);
-                    return Logger.Log(await SignInOrTwoFactorAsync(user, isPersistent));
-                }
-                if (UserManager.SupportsUserLockout && shouldLockout)
+                return error;
+            }
+            if (await IsLockedOut(user))
+            {
+                Logger.LogWarning("User {userId} is currently locked out.", await UserManager.GetUserIdAsync(user));
+                return SignInResult.LockedOut;
+            }
+            if (await UserManager.CheckPasswordAsync(user, password))
+            {
+                await ResetLockout(user);
+                return await SignInOrTwoFactorAsync(user, isPersistent);
+            }
+            if (UserManager.SupportsUserLockout && shouldLockout)
+            {
+                // If lockout is requested, increment access failed count which might lock out the user
+                await UserManager.AccessFailedAsync(user);
+                if (await UserManager.IsLockedOutAsync(user))
                 {
-                    // If lockout is requested, increment access failed count which might lock out the user
-                    await UserManager.AccessFailedAsync(user);
-                    if (await UserManager.IsLockedOutAsync(user))
-                    {
-
-                        return Logger.Log(SignInResult.LockedOut);
-                    }
+                    return await LockedOut(user);
                 }
-                return Logger.Log(SignInResult.Failed);
             }
+            // REVIEW: do we want to log normal password failures which would show up here?
```

Comment:
Maybe @blowdart will have an opinion. I think it is fine with logging just lockouts.